src/command_modules/azure-cli-vm/azure/cli/command_modules/vm/commands.py

```diff
@@ -0,0 +1,152 @@
+﻿#---------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+#---------------------------------------------------------------------------------------------
+
+from importlib import import_module
+
+from azure.cli.core.commands import DeploymentOutputLongRunningOperation, cli_command
+
+from azure.cli.core.commands.arm import cli_generic_update_command
+from azure.mgmt.compute.operations import (
+    VirtualMachinesOperations,
+    ContainerServiceOperations)
+
+from azure.cli.command_modules.vm._client_factory import *
+
+# VM
+
+cli_command(__name__, 'vm create', 'azure.cli.command_modules.vm.mgmt_vm.lib.operations.vm_operations#VmOperations.create_or_update', cf_vm_create,
+                         transform=DeploymentOutputLongRunningOperation('Starting vm create'))
+
+cli_command(__name__, 'vm delete', 'azure.mgmt.compute.operations.virtual_machines_operations#VirtualMachinesOperations.delete', cf_vm)
```

Comment:
It would be nice if we could clean these up, i.e. extract the common parts of the strings into some kind of helper method. Then again, it might just produce something that isn't noticeably shorter, just different...

Overall I like this much better. If the dev put a typo in the string, would it still error out when they tried to run a command? In other words, would they still get early warning of a problem, or would it only throw if they tried to run THAT specific command?  If so, I would have greater concern about the potential of typos in these strings.